fenix/app/src/main/java/org/mozilla/fenix/wifi/WifiConnectionMonitor.kt

```diff
@@ -30,7 +30,6 @@ class WifiConnectionMonitor(app: Application) {
     @VisibleForTesting
     internal val callbacks = mutableListOf<(Boolean) -> Unit>()
 
-    @VisibleForTesting
     internal var connectivityManager = app.getSystemService(Context.CONNECTIVITY_SERVICE) as
```

Comment:
Just double checking if this change was required/expected.

Reply:
I did this because I call connectivityManager in DownloadLanguagesPreferenceFragmentIntegration and I don't want to create a new object.

Reply:
Thanks for the additional info! 

I'm not sure what the best practices are here. Just tagging @gabrielluong for visibility too on this one, in case there is a reason not to.

Reply:
Let's avoid doing this. We should instead request the connectivity service in the feature and release it when we don't need it. Please follow `BiometricPrompt` in [BiometricPromptFeature](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/settings/biometric/BiometricPromptFeature.kt), we have request the system service (BiometricPrompt) from the activity context and release it after, if you're looking for an example towards your decision. Thank you to @jonalmeida for pointing this out.

Reply:
Done